ai_diffusion/api.py

```diff
@@ -64,7 +63,12 @@ class CheckpointInput:
     self_attention_guidance: bool = False
     dynamic_caching: bool = False
     tiled_vae: bool = False
-
+    magcache_enabled: bool = False
+    magcache_thresh: float = 0.24  
```

Comment:
Please remove the parameters also from here and `settings.py`.

They can be dug up from git if ever needed, but I'd rather not have them in the code without a good reason.

And if more configurability is desired at some point, a single "strength" value or some preset would be a more user friendly way to control it anyway.